kernel/environ.mli

```diff
@@ -145,7 +145,7 @@ val named_context_of_val : named_context_val -> Constr.named_context
 val val_of_named_context : Constr.named_context -> named_context_val
 val empty_named_context_val : named_context_val
 val ids_of_named_context_val : named_context_val -> Id.Set.t
-
+val ids_of_named_context_val_mem : named_context_val -> Id.t -> bool
```

Comment:
The function name doesn't follow the conventions from that file, and furthermore it doesn't match its actual behaviour. I'd rather call it `mem_var_val`, and similarly for the environ-taking variant, `mem_var`.

Reply:
(Also, arguments are flipped w.r.t. the standard convention, see e.g. `mem_constant.)

Reply:
I'll do it thanks.
To be sure, you mean `mem_var_val : named_context_val -> Id.t -> bool` or
`mem_var_named_context_val : named_context_val -> Id.t -> bool`

Reply:
The first one.